src/reports/ChartReportPage.jsx

```diff
@@ -123,7 +141,8 @@ const ChartReportPage = () => {
               }}
             >
               <XAxis
-                dataKey="fixTime"
+                dataKey={timeIndex}
+                interval="preserveStartEnd"
```

Comment:
What does this do?

Reply:
I had an issue with labels overlapping, but im struggling to reproduce it now. Here it is on a tab i still had open:
![image](https://github.com/traccar/traccar-web/assets/14150258/9fefffb0-050d-4370-8445-4bdea49788de)
```
interval="preserveStartEnd"
```
Causes overlapping labels to be hidden/grouped, when the default behavior is to show them all. `preserveStartEnd` just makes sure that the label at the start/end of the axis are not hidden, so that the timestamp where the graph starts and ends is preserved

Reply:
Oh, i can reproduce it now. Had to clear browser cache. This is what it looks like without preserving start/end
![image](https://github.com/traccar/traccar-web/assets/14150258/aa7e6d05-01f2-4129-9580-6f83863ef6bc)

Reply:
Can you provide a comparison? I'm not sure I understand what it actually does.

Reply:
Yep, but I actually think I should remove this line and address it in a separate issue, because I think it needs further investigation. It's also not directly related to this PR, so I might remove this line and go over it in a new issue for someone to look into further later.

[interval](https://recharts.org/en-US/api/XAxis#interval) has a few options, but the documentation is really lacking. It can be a number, but its not clear what the number represents (i assume every X tick is used as a label). Setting it to 0, will apparently show all ticks on the X axis. So if you have 1000 ticks, you have 1000 labels, and they will all overlap and be illegible.

A value of `"preserveStart"`, `"preserveEnd"` or `"preserveStartEnd"`, is supposed to result in an automatic calculation of which labels to show, so that none of them overlap. Basically if you have 1000 labels, it will hide most of them, with conditions.

The default value is "preserveEnd". This should hide all the overlapping labels, but prioritize keeping the last one
![image](https://github.com/traccar/traccar-web/assets/14150258/6278347a-51ff-4749-9a18-94e61e6787e7)

"preserveStart" should hide all the overlapping labels, but prioritize keeping the first one
![image](https://github.com/traccar/traccar-web/assets/14150258/e206ce23-c970-4bdf-a237-58d261e665a3)

"preserveStartEnd" is just a combination of both. it makes sure that the labels at either end of the axis are displayed, and any that are hidden are somewhere in between. Except in testing, it doesn't always, and often you get the same problem of labels overlapping
![image](https://github.com/traccar/traccar-web/assets/14150258/475724cc-7ea6-4864-bc3e-eedd22aa2f3a)

Initially when I tested this, I had the following:
![image](https://github.com/traccar/traccar-web/assets/14150258/61e32645-65c1-4cf1-9514-fdbff1cec76a)

And then after changing the interval to preserveStartEnd, i had this:
![image](https://github.com/traccar/traccar-web/assets/14150258/a43ab607-eba2-4971-9395-326c024e0774)

So I thought i had rectified the issue and moved on. Basically a "this isnt related but i'll fix that bug while im here".
But in further testing, the actual result seems to be unpredictable. The value likely should be preserveStartEnd, and that should be the [recharts ](https://recharts.org/) default, but it isn't, and changing it doesn't actually fix the issue.

More likely than not, this is actually a bug in recharts. A lot of people seem to be having issues with these labels. Interval should honor the value of [minTickGap](https://recharts.org/en-US/api/XAxis#minTickGap), which by default is supposed to ensure a minimum gap between the labels, but in practice it just doesn't work the way the docs say it does. It seems like there is some kind of disconnect between `interval`, `minTickGap`, and `tickFormatter`. Other people have come up with [convoluted fixes for this issue](https://github.com/recharts/recharts/issues/3305#issuecomment-1427133689), but really it comes down to recharts behavior.

Anyway, happy to remove this line and investigate it separately. It doesn't help, so its really pointless being here